.github/workflows/e2e.yaml

```
@@ -21,7 +21,7 @@ jobs:
     - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
     - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
       with:
-        go-version: '1.20'
+        go-version: '1.21'
```

```suggestion
        go-version-file: 'go.mod'
```

This keeps it up to date as we update `go.mod`, and prevents future update PRs like this.